### ISSUE TITLE
image service suggestion

### ISSUE THREAD

**Issue by saiqulhaq** [loqimean/fake_picture#1]
https://github.com/loqimean/fake_picture/blob/06c6f24780d581ee1809e850c01682e1bf6819c7/lib/fake_picture.rb#L15

what if using [https://loremflickr.com](https://loremflickr.com) or [https://picsum.photos/](https://picsum.photos/)  instead?
example:
* https://loremflickr.com/g/300/300/all
* https://picsum.photos/200/300

**Comment by saiqulhaq** [comment]
related gem https://discuss.rubyonrails.org/t/download-random-images-from-unspash-or-show-them-with-a-helper/80817

**Comment by loqimean** [comment]
Yeah, thank you, nice idea, may be you wanna contribute this feature?)

**Comment by saiqulhaq** [comment]
if it's ok to use this gem https://github.com/railsjazz/unsplash_image
then I will do it
